README.md

```diff
@@ -76,17 +81,17 @@ If you use vim, just start with the numbers and alphabet, otherwise look at gene
 ### Alphabet
```

Comment:
Why did you include the `.git` here? I would have just done a find-replace to community without the `.git`

Reply:
You are quite right. Fixed.

Reply:
For HTTPS cloning, GitHub actually recommends `https://github.com/talonhub/community.git` for the HTTPS URL, but it works without the `.git`  and the old way did not include the `.git`, so I removed it, but not sure what the best way is.